### ISSUE TITLE
Update to elm 0.17

### ISSUE THREAD

**Issue by tiziano88** [deadfoxygrandpa/elm-test#44]

**Comment by tiziano88** [comment]
Blocked by laszlopandy/elm-console#12 .

**Comment by bdukes** [comment]
I believe this repo has migrated to https://github.com/elm-community/elm-test/tree/1.1.0, which supports 0.17

**Comment by tiziano88** [comment]
Thanks @bdukes ! Though it seems that running tests from command line is also blocked by the same issue even for the elm-community fork of this package.

As a side note, @deadfoxygrandpa , should we add a notice to this repo to redirect users to the elm-community fork, or do you plan to keep maintaining this version anyway going forward?

**Comment by Anahkiasen** [comment]
I think you guys should put up a deprecation notice. As a newcomer to Elm it confused me that the repositories had the same name with this one having the most stars, but the other one being the actual official one

**Comment by janwirth** [comment]
Yeah so this also happened to me, @deadfoxygrandpa I think you are saving a lot of time for new devs if you add a deprecation note.

Please merge my PR.
https://github.com/deadfoxygrandpa/elm-test/pull/46
